### ISSUE TITLE
loss keeps 0 during training (clustering)

### ISSUE THREAD

**Issue by z94007** [XifengGuo/DEC-keras#5]
During training with clustering layers, the loss keeps 0 through all the iteration while nmi, acc and ari work fine. Why does this happen? Does it indicate that the encoder layers haven't been trained during training?
